### DISCUSSION TITLE
Kafka connector UI via Conduktor/Kafka UI

### DISCUSSION THREAD

**Discussion by fathimaSheikh** [strimzi/strimzi-kafka-operator#8543] [Q&A]
We previously had a Kafka connect cluster to which we could connect using the URL in Conduktor or Kafka UI. 
Wondering if this is possible with Strimzi operated Kafka connect. If so, how can I do that?

**Comment by scholzj** [comment]
I think that in general it should work. But it might depend on many small details. E.g. where does your Conduktor run (I know it used to be a desktop app but I'm not sure it is still the case - I think it might not) as you might need to for example expose the Connect service to make it accessible etc. 

Also, please keep in mind that when you enable the `KafkaConnector` resources for your Connect cluster, you can use the REST API only for monitoring - any changes you do through it would be reverted. If it is disabled, you can do anything over the REST API without any restrictions.

**Reply by fathimaSheikh** [reply]
Hi @scholzj , Thanks for quick response.
I have disabled the `KafkaConnector` resources and not depending on API for anything.

I have Conduktor desktop application. Will try to expose the connect service and share an update.

**Comment by mozarik** [comment]
Hi @scholzj and @fathimaSheikh  i also want to use Kafka Connector UI via Kafka UI. But right know i'm encountring an issue that i cannot access the Kafka Connect tab on the UI

I following the suggestiong here https://github.com/strimzi/strimzi-kafka-operator/issues/3201 but it still cannot connected it got 500 HTTP status error code

And then i try to hit it inside the kafka-ui pod i got this response 

```
/usr/bin $ wget http://my-connect-cluster-connect-api:8083/connectors
Connecting to my-connect-cluster-connect-api:8083 (***.***.***.***:8083)
wget: can't open 'connectors': Permission denied
```
